shared/common-adapters/button.desktop.js

```diff
@@ -31,11 +118,18 @@ Button.propTypes = {
   label: React.PropTypes.string,
   style: React.PropTypes.object,
   primary: React.PropTypes.bool,
+  secondary: React.PropTypes.bool,
```

Comment:
can you import the type from flow and nuke the prop types definition so we don't have duplicate defs?